### PR TITLE
RavenDB-6967 Clear Index Errors - Fixed

### PR DESCRIPTION
**Pull request by Danielle9897**

### CODE REVIEW

File: src/Raven.Studio/typescript/commands/database/index/clearIndexErrorsCommand.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import database = require("models/resources/database");
+import endpoints = require("endpoints");
+
+class clearIndexErrorsCommand extends commandBase {
+    constructor(private indexesNames: string[], private db: database) {
+        super();
+    }
+
+    execute(): JQueryPromise<void> {
+        const args =  this.indexesNames ? { name: this.indexesNames } : "";
```
Comment (ml054) [inline src/Raven.Studio/typescript/commands/database/index/clearIndexErrorsCommand.ts]: const args =  this.indexesNames ? { name: this.indexesNames } : {}.

File: src/Raven.Studio/typescript/commands/database/index/clearIndexErrorsCommand.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        const url = endpoints.databases.index.indexesErrors + this.urlEncodeArgs(args);
+
+        return this.del<void>(url, null, this.db)
+            .done(() => this.reportSuccess(`Successfully cleared errors from ${this.indexesNames ? 'selected' : 'all'} indexes`));
```
Comment (ml054) [inline src/Raven.Studio/typescript/commands/database/index/clearIndexErrorsCommand.ts]: I wouldn't assume here that we removed selected from this command perspective. If you invoke this method from other place, like edit index view then this text won't be correct.

Comment (Danielle9897) [reply src/Raven.Studio/typescript/commands/database/index/clearIndexErrorsCommand.ts]: done

File: src/Raven.Studio/typescript/commands/database/index/clearIndexErrorsCommand.ts (hunk 1 above)
Comment (Danielle9897) [reply src/Raven.Studio/typescript/commands/database/index/clearIndexErrorsCommand.ts]: done

**Review by ml054 — APPROVED** [review]

**Pull request merged**